bindings/kepler.gl-jupyter/js/lib/keplergl/components/panel-header.js

```diff
@@ -22,11 +22,14 @@ import {PanelHeaderFactory, Icons, withState} from 'kepler.gl/components';
 import {toggleModal} from 'kepler.gl/actions';
 import React from 'react';
 import {IntlProvider} from 'react-intl';
+import {visStateLens} from 'kepler.gl/reducers';
+import {exportJson} from 'kepler.gl/dist/utils/export-utils'
```

Comment:
you should add export `exportJson` from `utils/index.js` them import it here with
`import {exportJson} from 'kepler.gl/utils'`

Reply:
None of `kepler.gl/utils` is exported directly. It's missing from the npm [files](https://github.com/keplergl/kepler.gl/blob/master/package.json#L76). Should I add that too?

Reply:
urr right, it needs to be add to `package.json` `"files"` field

Reply:
I can do that and cut a new release

Reply:
Thanks. Can you update the version the jupyter binding uses at the same time? It's still on `2.4.0`.